crates/evm/src/executor/inspector/cheatcodes/env.rs

```diff
@@ -395,8 +395,10 @@ pub fn apply<DB: DatabaseExt>(
             trace!(address=?inner.0, code=?hex::encode(&code), "etch cheatcode");
             // TODO: Does this increase gas usage?
             data.journaled_state.load_account(h160_to_b160(inner.0), data.db)?;
-            data.journaled_state
-                .set_code(h160_to_b160(inner.0), Bytecode::new_raw(code.0).to_checked());
+            data.journaled_state.set_code(
+                h160_to_b160(inner.0),
```

Comment:
I am realizing some of this code snafu could've been avoided if we had added a `From<ethers::U256> for alloy::U256` etc. compatibility layer in Alloy, to make the migration easier - as an optional feature, and then remove it

Reply:
Yep. We'll introduce a conversion trait on foundry to aid with this after we merge this PR!